script/load.bety.sh

```diff
@@ -404,7 +428,27 @@ for T in ${EMPTY_TABLES} ${CLEAN_TABLES} ${CHECK_TABLES} ${MANY_TABLES}; do
     echo "SELECT COUNT(*) FROM ${T};" >&3 && read START <&4
     if [[ "${EMPTY}" == "NO" || ${EMPTY_TABLES} == *"$T"* ]]; then
       if [ -f "${DUMPDIR}/${T}.csv" ]; then
-        echo "\COPY ${T} FROM '${DUMPDIR}/${T}.csv' WITH (DELIMITER '	',  NULL '\\N', ESCAPE '\\', FORMAT CSV, ENCODING 'UTF-8')" >&3
+	T_SIZE=`cat ${DUMPDIR}/${T}.csv | wc -l`
+	if [ "${T_SIZE}" -gt "${SPLIT_FILE_MAX}" ]; then
```

Comment:
can we have SPLIT_FILE_MAX==0 means use the old algorithm, and for that to be the default value?